strawberryfields/apps/similarity.py

```diff
@@ -591,11 +591,12 @@ def feature_vector_orbits(
         graph (nx.Graph): input graph
         list_of_orbits (list[list[int]]): a list of orbits
         n_mean (float): total mean photon number of the GBS device
-        samples (int): number of samples used in the Monte Carlo estimation
+        samples (int): optional number of samples used in the Monte Carlo estimation. Defaults to exact calculation if ``samples`` is unspecified.

```

Comment:
This line may be too long

Reply:
I made it two lines and checked that the documents are building correctly